x/oracle/types/genesis.go

```diff
@@ -12,7 +12,7 @@ func NewGenesisState(
 	feederDelegations []FeederDelegation, missCounters []MissCounter,
 	aggregateExchangeRatePrevotes []AggregateExchangeRatePrevote,
 	aggregateExchangeRateVotes []AggregateExchangeRateVote,
-	TobinTaxes []TobinTax,
+	tobinTaxes []TobinTax,
```

Comment:
this is the difference between a public and private variable. tobinTaxes is in fact camelCase